### ISSUE TITLE
Grammar::Tracer broken on new Rakudo

### ISSUE THREAD

**Issue by moritz** [jnthn/grammar-debugger#2]
```
$ perl6 --version
This is perl6 version 2012.11-107-g70d586f built on parrot 4.10.0 revision RELEASE_4_10_0-2-g2e66846
$ perl6 -e 'use Grammar::Tracer; grammar A { rule TOP { "foo" } }; A.parse(" foo")'
TOP
|  ws
|  orig
No such method 'MATCH' for invocant of type 'Str'
  in block  at lib/Grammar/Tracer.pm:20
  in method MATCH at src/gen/CORE.setting:10455
  in block  at lib/Grammar/Tracer.pm:20
  in regex TOP at -e:1
  in block  at lib/Grammar/Tracer.pm:16
  in method parse at src/gen/CORE.setting:10538
  in block  at -e:1
```

**Comment by hoelzro** [comment]
I just tried this, and got the following:

    $ perl6 -e 'use Grammar::Tracer; grammar A { rule TOP { "foo" } }; A.parse(" foo")'
    TOP
    * FAIL

So it's not blowing up, and it's supposed to fail because leading and trailing whitespace in a rule doesn't count for `<.ws>`, does it?  Right @meisl and @jnthn?

**Comment by meisl** [comment]
Exactly.
It's long been fixed and can - as well as #8 - be closed... ;-)
